expression/builtin_time.go

```diff
@@ -1661,6 +1661,10 @@ func (c *fromUnixTimeFunctionClass) getFunction(ctx sessionctx.Context, args []E
 		return nil, err
 	}
 
+	if fieldString(arg0Tp.GetType()) {
```

Comment:
why not use `isArg0Str`?

Reply:
TypeUnspecified is not the expected type，which satisfies types.IsString.
and TypeVarString is an input parameter, no need to adjust the precision .